milestone2.ts

```diff
@@ -83,35 +83,51 @@ const userData: Volunteer[] = [
   Hint: user data is stored in the userData object above. 
 */
 type GetNumber = (data: Volunteer[]) => number;
-let findAverage: GetNumber; // Define lambda function here
+let findAverage: GetNumber = (data) => {
+  let total = 0;
+  for (let volunteer of data) {
+    total += volunteer.age;
+  }
+  return total / data.length;
+}
 
-//console.log(findAverage(userData));
+console.log(findAverage(userData));
 
 // Question 2: Data Handling
 // Please AVOID USING FOR LOOPS for questions 2 through 4
 /* Use data handling function(s) to find the first index of someone from San Francisco (SF).
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+let isFromSf = (volunteer: Volunteer) => {
+  return volunteer.city === "SF";
+}
 
-//console.log(findIndexAns(userData));
+let findIndexAns: GetNumber = (data) => {
+  return data.findIndex(isFromSf);
+}
```

Comment:
would be more readable and concise to just define the lambda function inside the findIndex call as you did with the other questions.